packages/hash/api/src/db/postgres/link/sql/link_versions.util.ts

```diff
@@ -27,23 +26,6 @@ export type DbLinkVersionRow = {
   updated_by_account_id: string;
 };
 
-export const selectAllLinkVersions = sql<DbLinkRow>`
-  select ${linkVersionsColumnNamesSQL}
-  from links
-`;
-
-export const selectAllLinkVersionsOfLink = (params: {
```

Comment:
Is the removal of this related to linked aggregations, or is it some left-over utility functions from another refactor that aren't used anymore?

Reply:
These are unrelated to this PR yes, but I noticed whilst working on it that these are unused methods so thought I might as well remove them.